test-app/tsconfig.json

```diff
@@ -6,11 +6,24 @@
     // work with TypeScript.
     "baseUrl": ".",
     "paths": {
-      "test-app/tests/*": ["tests/*"],
-      "test-app/*": ["app/*"],
-      "*": ["types/*"]
+      "test-app/tests/*": [
+        "tests/*"
+      ],
+      "test-app/*": [
+        "app/*"
+      ],
+      "*": [
+        "types/*"
+      ]
     },
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "lib": [
```

Comment:
not sure if this should be added here, but it was the only way to get the errors to show properly when developing and adding tests.